distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogSearchUi.java

```diff
@@ -40,244 +57,432 @@
-                .when();
-    }
+    private static final DynamicPort CSW_STUB_SERVER_PORT = new DynamicPort(6);
 
-    private static Map<String, String> makePermission(String attribute, String action,
-            String value) {
-        return ImmutableMap.of("attribute", attribute, "action", action, "value", value);
-    }
+    public static final DynamicUrl CSW_STUB_SERVER_PATH = new DynamicUrl(INSECURE_ROOT,
+            CSW_STUB_SERVER_PORT,
+            "/services/csw");
 
-    private static void delete(String id) {
-        asAdmin().expect()
-                .log()
-                .all()
-                .statusCode(200)
-                .when()
-                .delete(api() + "/" + id);
+    private static final String CSW_STUB_SOURCE_ID = "cswStubServer";
+
+    private static FederatedCswMockServer cswServer;
+
+    private static final int CSW_SOURCE_POLL_INTERVAL = 10;
+
+    private static final String POLL_INTERVAL = "pollInterval";
+
+    @BeforeExam
+    public void beforeExam() throws Exception {
+        try {
+            waitForSystemReady();
+            cswServer = new FederatedCswMockServer(CSW_STUB_SOURCE_ID,
+                    INSECURE_ROOT,
+                    Integer.parseInt(CSW_STUB_SERVER_PORT.getPort()));
+            cswServer.start();
+
+            Map<String, Object> cswStubServerProperties = getCswTransactionalSourceProperties(
+                    CSW_STUB_SOURCE_ID,
+                    CSW_STUB_SERVER_PATH.getUrl(),
+                    getServiceManager());
+            cswStubServerProperties.put(POLL_INTERVAL, CSW_SOURCE_POLL_INTERVAL);
+            getServiceManager().createManagedService(CSW_TRANSACTIONAL_SOURCE_FACTORY_PID,
+                    cswStubServerProperties);
+
+            getServiceManager().waitForHttpEndpoint(CSW_PATH + "?_wadl");
+            getServiceManager().waitForHttpEndpoint(WORKSPACE_URL.getUrl());
+
+            getCatalogBundle().waitForFederatedSource(CSW_STUB_SOURCE_ID);
+            getServiceManager().waitForSourcesToBeAvailable(REST_PATH.getUrl(), CSW_STUB_SOURCE_ID);
+
+            LOGGER.info("Source status: \n{}", get(REST_PATH.getUrl() + "sources").body()
+                    .prettyPrint());
+        } catch (Exception e) {
+            LoggingUtils.failWithThrowableStacktrace(e, "Failed in @BeforeExam: ");
+        }
     }
 
     @Test
     public void testGuestCantCreateWorkspace() throws Exception {
-        Map<String, String> workspace = ImmutableMap.of("title", "my workspace");
-        expect(asGuest().body(stringify(workspace)), 404).post(api());
+        Map<String, String> workspace = ImmutableMap.of(Core.TITLE, "my workspace");
+        expect(asGuest().body(stringify(workspace)), 404).post(workspaceApi());
     }
 
     @Test
     public void testGuestCanCreateWorkspacesForOthers() {
-        Map<String, String> workspace = ImmutableMap.of("title",
+        Map<String, String> workspace = ImmutableMap.of(Core.TITLE,
```

Comment:
I don't think @djblue meant the actual attribute title. The attributes coming in on the workspace json do not always reflect what they get transformed to for backend attributes.
So its technically not wrong, but it also just "happens" to match. May never change, but it is slightly misleading.
See WorkspaceTransformer L71 - L99 for the transforms that happen between the json and the metacard attribute names.